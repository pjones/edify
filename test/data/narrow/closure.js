let makeCounter = function(startingValue) {
  let n = startingValue;

  // <<: closure
  return function() {
    return n += 1;
  };
  // :>>
};
