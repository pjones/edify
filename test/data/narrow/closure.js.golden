return function() {
  return n += 1;
};
